core/libraries/plugin_api/EE_Register_Addon.lib.php

```diff
@@ -322,7 +331,7 @@ private static function _verify_parameters($addon_name, array $setup_args)
             );
         }
         // check that addon has not already been registered with that name
-        if (isset(self::$_settings[ $addon_name ]) && ! did_action('activate_plugin')) {
+        if (isset(self::$_settings[$addon_name]) && ! did_action('activate_plugin')) {
```

Comment:
our code sniffer expects spaces around variables used as array keys
plz see:
https://travis-ci.org/eventespresso/event-espresso-core/jobs/376461099

This can be adjusted in your IDE at:
phpStorm Settings > Editor > Code Style > PHP > Spaces tab > Within section > Brackets around variable/expression

Reply:
taken care of in the privacy data exporter PR